README.md

```diff
@@ -19,6 +19,22 @@ certificate-transparency: Auditing for TLS certificates
  - [Deploying a Log](#deploying-a-log)
  - [Operating a Log](#operating-a-log)
 
+C++ Log Server Deprecation Notice
+---------------------------------
+
+The CT log server implementation in this repository is no longer under
+active development. We recommend that new deployments use the new Go
+based server, which can handle much larger Merkle trees:
+
+[CT Personality](https://github.com/google/certificate-transparency-go)
+[Generic Backend](https://github.com/google/trillian)
+
+For more information on why we recommend this see the
+[Deprecation Notes](DEPRECATED.md)
+
+N.B. This notice refers to the servers only, not the other client code in this
+repository.
+
```

Comment:
Could we move this section below the Introduction, and add a cross-reference to it from the first bullet of the introduction?
```
...The main areas covered are:
 - (**Deprecated**, see [below](#c++-log-server-deprecation-notice)). An open-source, distributed, implementation of a CT Log server, also including:
     - An implementation of a read-only ["mirror" server](docs/MirrorLog.md)
       that mimics a remote Log.
     - Ancillary tools needed for managing and maintaining the Log.
```

Might also be worth shuffling the bullets in the Introduction so the deprecated server is listed last.

Reply:
Done.